radio/src/gui/128x64/lcd.cpp

```diff
@@ -534,8 +559,7 @@ void lcdDrawNumber(coord_t x, coord_t y, lcdint_t val, LcdFlags flags, uint8_t l
   lcdLastPos = x;
   x -= fw;
   if (dblsize) x++;
-
-  for (uint8_t i=1; i<=len; i++) {
+    for (uint8_t i=1; i<=len; i++) {
```

Comment:
Indent level.  Good example of why at least a linebreak after any `if` is really useful.